app/src/main/java/sdp/moneyrun/LeaderboardListAdapter.java

```diff
@@ -0,0 +1,39 @@
+package sdp.moneyrun;
+
+import android.app.Activity;
+import android.view.LayoutInflater;
+import android.view.View;
+import android.view.ViewGroup;
+import android.widget.ArrayAdapter;
+import android.widget.TextView;
+
+import java.util.ArrayList;
+
```

Comment:
Maybe I am just not understanding the full picture, but why do we need an adapter? Can't we just do everything inside Leaderboard activity?

Reply:
Because this is how simple and efficient implementations of ListView works, it is also easier to work with  for the person who will link it with the database